doc/dev/developer_guide/testing_integration_tests/tests-integration-testing-teuthology-workflow.rst

```diff
@@ -200,9 +198,9 @@ much.
 Killing Tests
 -------------
 Sometimes a teuthology job might not complete running for several minutes or
-even hours after tests that were trigged have completed running and other
-times wrong set of tests can be triggered is filter wasn't chosen carefully.
-To save resource it's better to termniate such a job. Following is the command
+even hours after tests that were trigged have completed running. Other
+times wrong set of tests can be triggered if a filter wasn't chosen carefully.
+To save resources it's better to terminate such jobs. Here is the command
```

Comment:
Let's simply the wording here.

Teuthology-kill can be used to kill jobs that have been running unexpectedly for several hours or when developers want to intentionally terminate tests before they complete.

Reply:
I can do this in this PR or in another PR that's in the queue right now. We can discuss this in the morning, at our fortnightly meeting.

Reply:
Well, I ended up fixing this here. So: qapla, I guess.